src/main/java/org/fundacionjala/coding/nestor/ChildrenMovie.java

```diff
@@ -0,0 +1,18 @@
+package org.fundacionjala.coding.nestor;
+
+public class ChildrenMovie extends Movie {
+    private static double PRICE = 1.5;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "PRICE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)